.pre-commit-config.yaml

```diff
@@ -45,11 +45,10 @@ repos:
             language_version: '3.10'
             additional_dependencies:
                 - click==8.1.3
-                - fastapi==0.75.2
-                - pydantic==1.9.0
-                - pytest==7.1.2
-                - types-requests==2.27.27
-                - types-ujson==4.2.1
+                - fastapi<=0.95, >0.75
```

Comment:
Why not relax the requirements and define only a boundary for the major version?

Reply:
I would assume at the same major, all should be backwards compatible, but seeing that `fastapi` is currently by a minor==95, I am not sure this can be assured.

Reply:
But why not only leave >0.75?